docs/WHY.md

```diff
@@ -26,6 +26,78 @@ Mirascope addresses these pain points with a philosophy centered on transparency
     - **Type Safety**: Catch errors before runtime with our comprehensive type annotations.
     - **Intuitive API**: Our API is designed to be predictable and easy to learn, following Python best practices.
 
+## Comparison with Official SDKs
```

Comment:
I think the biggest thing here to cover is tools and agents at least in some capacity. that's really where the simplified syntax of mirascope + convenience functions really shine (and reduce code without reducing transparency).

I think maybe drop the third point about "Streaming Support" and include streaming, tools, and agents as examples of the first point on simplified syntax?